.github/workflows/CI.yml

```diff
@@ -40,12 +40,23 @@ jobs:
       run: ./build ci
       if: ${{ runner.os != 'Windows' }}
 
-    - name: Test - Persistence
-      run: dotnet bin/Debug/Persistence.Tests/Persistence.Tests.dll
+    - name: Test - YamlValidator
```

Comment:
could we move the validator tests to come after the core output libraries get tested?
If the base assemblies fail, then there may be no reason to also run tests on clients that depend on those libraries.